### PR TITLE
update algorithm

### PR DESCRIPTION
**Pull request by MarianBastiurea**

### CODE REVIEW

**Review by andreiursudev — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -69,11 +69,46 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
-
+        <dependency>
```
Comment (andreiursudev) [inline pom.xml:76]: Why do you need this dependency? Remove if not needed.

File: pom.xml [hunk 2]
```diff
@@ -69,11 +69,46 @@
...
+            <groupId>jakarta.servlet.jsp.jstl</groupId>
+            <artifactId>jakarta.servlet.jsp.jstl-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.web</groupId>
+            <artifactId>jakarta.servlet.jsp.jstl</artifactId>
+        </dependency>
+
+        <dependency>
```
Comment (andreiursudev) [inline pom.xml:104]: Why do you need this dependency? Remove if not needed.

File: pom.xml [hunk 3]
```diff
@@ -69,11 +69,46 @@
...
+            <groupId>org.webjars</groupId>
+            <artifactId>bootstrap</artifactId>
+            <version>5.1.3</version>
+        </dependency>
+
+        <dependency>
+            <groupId>org.webjars</groupId>
```
Comment (andreiursudev) [inline pom.xml:111]: Why do you need this dependency? Remove if not needed.

File: pom.xml [hunk 4]
```diff
@@ -69,11 +69,46 @@
...
+            <artifactId>jquery</artifactId>
+            <version>3.6.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.webjars</groupId>
+            <artifactId>bootstrap-datepicker</artifactId>
```
Comment (andreiursudev) [inline pom.xml:117]: Why do you need this dependency? Remove if not needed.

**Review by andreiursudev — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (MarianBastiurea) [reply pom.xml:76]: Trainer add this dependency

File: pom.xml (hunk 2 above)
Comment (MarianBastiurea) [reply pom.xml:104]: Trainer add this dependency

File: pom.xml (hunk 3 above)
Comment (MarianBastiurea) [reply pom.xml:111]: Trainer add this dependency. Needed it.

File: pom.xml (hunk 4 above)
Comment (MarianBastiurea) [reply pom.xml:117]: Trainer add this dependency. Needed it.

**Review by andreiursudev — APPROVED** [review]

**Pull request merged**